src/raftstore/store/store.rs

```diff
@@ -511,6 +515,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             self.snap_mgr.clone(),
             self.cfg.snap_apply_batch_size.0 as usize,
             self.cfg.use_delete_range,
+            end_point_request_max_handle_secs + self.cfg.range_deletion_delay.as_secs(),
```

Comment:
It means delay = 120s + 60s?

Reply:
60 + 60, `range deletion delay` is 60.

Reply:
Maybe just use `end_point_request_max_handle_secs * 2`, I think we will rarely configure this value.

Reply:
You can precalculate and adjust the `range_deletion_delay` after loading the configuration file, don't need to pass the `end_point_request_max_handle_secs` inside.